packages/react/src/alert-dialog/AlertDialog.tsx

```diff
@@ -0,0 +1,89 @@
+import * as RadixAlertDialog from "@radix-ui/react-alert-dialog";
+import { type ComponentPropsWithRef, type ReactNode, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Box } from "../box";
+import { Button } from "../button";
+import { Flex } from "../flex";
+import * as styles from "./AlertDialog.css";
+
+type AlertDialogProps = ExtendProps<
+  ComponentPropsWithRef<typeof RadixAlertDialog.Root>,
+  ComponentPropsWithRef<typeof Box>,
+  {
+    action?: string;
+    actionAppearance?:
+      | "danger"
+      | "danger-outline"
+      | "default"
+      | "primary"
+      | "secondary";
+    cancel?: string;
+    children: ReactNode;
+    onAction: () => void;
+    onCancel: () => void;
+    title: ReactNode;
+  } & styles.DialogVariants
+>;
+
+export const AlertDialog = forwardRef<HTMLDivElement, AlertDialogProps>(
+  (
+    {
+      action = "Confirm",
+      actionAppearance = "primary",
+      cancel = "Cancel",
+      children,
+      onAction,
+      onCancel,
+      open,
+      size = "md",
```

Comment:
let's make `sm` the default - for alert dialogs typically that is the default size